doc/RecordingControl.xml

```diff
@@ -2221,9 +2221,58 @@ secfrac = "." 1*6DIGIT</programlisting>
+            <row>
+              <entry>
+                <para>
+                  <literal>Timescale</literal>
+                </para>
+              </entry>
+              <entry>
+                <para>The <literal>timescale</literal> (or clock) from the segment in hertz</para>
+              </entry>
+              <entry>
+                <para>Integer (See <xref linkend="_refTimescaleAttribute"/>)</para>
+              </entry>
+            </row>
+            <row>
+              <entry>
+                <para>
+                  <literal>SeekTable</literal>
+                </para>
+              </entry>
+              <entry>
+                <para>Seek table to efficiently locate file fragments and seek in the file</para>
+              </entry>
+              <entry>
+                <para>String (See <xref linkend="_refSeekTableAttribute"/>)</para>
+              </entry>
+            </row>
           </tbody>
         </tgroup>
       </table>
+      <section xml:id="_refTimescaleAttribute">
+        <title>Timescale Attribute</title>
+        <para>
+          The value of the <literal>timescale</literal> property of the <literal>MediaHeaderBox</literal> in the segment as defined in ISO/IEC 14496-12:2012 section 8.4.2. If there are multiple tracks in the segment each of these boxes shall all have the same <literal>timescale</literal> value. The <literal>MovieHeaderBox</literal> shall also have the same <literal>timescale</literal> value.
+        </para>
+      </section>
+      <section xml:id="_refSeekTableAttribute">
+        <title>SeekTable Attribute</title>
+        <para>
+          This seek table attribute is a string that contains the 'mfra' (Movie Fragment Random Access) box corresponding to the associated ISO file. It is encoded in base64 with padding according to RFC 4658. The binary format of the box is defined in ISO/IEC 14496-12:2012 section 8.8.9. This optional box has to be the last one at the end of the file. It is copied in the object metadata to allow the receiver to efficiently locate file fragments and seek in the file without the need to access the file itself.
+        </para>
+        <para>
+          When the <literal>SeekTable</literal> attribute is present it shall contain at least one entry for the very first sample of each track. The device should attempt to include additional entries locating the first frame of each subsequent fragment but they could also be omitted to limit the size of the seek table. By definition, each entry can only reference a sample that is a random access point (ie: a keyframe). The device should avoid including more entries than those suggested here.
+        </para>
+        <figure xml:id="_refSeekTableStructure">
+          <title>Typical structure of a Movie Fragment Random Access ('mfra') box</title>
+          <mediaobject>
+            <imageobject>
+              <imagedata fileref="media/RecordingControl/seekTableStructure.svg" contentwidth="130mm"/>
```

Comment:
The Image is not rendering for the seekTableStructure.svg. All I see is the following;

![SeekTableStructure_Not_Rendering_Issue](https://github.com/user-attachments/assets/aba1a1f6-2703-4187-8e4d-7bad7056c1b0)

I believe what I should be seeing is the following file rendered;

![SeekTableStructure](https://github.com/user-attachments/assets/0b42398e-329c-4ff0-8299-687837a64b7f)

Reply:
You mentioned Internet Explorer? Is that a goal to be able to render in there? That's been discontinued/end-of-life/retired two years ago and removed from most Windows installations and I certainly did not expect to hear about it in 2024! 😄

Reply:
Ha!!, I took down the full PR and opened the recording xml in Microsoft Edge and rendered the file in compatibility mode. The same issue will happen when published on the ONVIF web site. It will need fixed :+1:

Reply:
I'm learning about the publishing process apparently, wasn't aware it relied on the IE engine :o I guess we could replace the SVG with a regular image if there's issue rendering that

Reply:
We don't actually render the xml/svg in the browser when we publish, as we ship the pdf on the onvif website. So I don't think this will be an issue if it renders properly in the generated PDF

Reply:
On a side note, its possible the contentwidth or some other property is badly set which triggers that time of rendering

Reply:
As mentioned by the colleagues,  rendering of the development pages requires a state of the art browser base like Chromium, Firefox or Safari. Edge is nowadays based on Chromium. Compatibility mode is no option.

Reply:
@HansBusch I use Edge and Chrome. 

To be reasonable and it is an expectation that all content of a specification inclusive of images should render properly regardless of what browser you use to review the content. 

I do agree with @jflevesque-genetec  that it should display properly in the PDF! Does it?

The primary point before it is missed by all that the image in the recordingcontrol.xml is not rendering as being shown in the screen shot and the images in the rest of the file do render correctly!!

Reply:
@kieran242 could you tell us how you render the page? Because on my side, it seems to render fine (and scales properly if I zoom in/out)
![image](https://github.com/user-attachments/assets/00caf07f-b526-4a1e-9010-11a3cf35ccac)